components/leave_channel_modal/leave_channel_modal.tsx

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Awesome stuff @sinansonmez! 

If possible, could you implement this as a functional component? We're making a push for new components to be functional rather than class-based. 

https://developers.mattermost.com/contribute/more-info/webapp/build-component/#implement-the-component

Reply:
@mylonsuren It is an existing component, I just renamed it. But you are right, the component is simple enough which can be converted to FC easily. I will do it asap